packages/entity-server/src/__tests__/__integration__/fixtures.ts

```diff
@@ -0,0 +1,307 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+type Entity = {
+  country_code?: string;
+  code: string;
+  name: string;
+  type: string;
+  attributes: {
+    type?: string;
+  };
+};
+
+export const getEntityWithFields = (entityCode: string, fields: (keyof Entity)[]) => {
+  const entity = ENTITIES.find(e => e.code === entityCode);
+
+  if (!entity) {
+    throw new Error(`No entity found with code ${entityCode} in test data`);
+  }
+
+  return Object.fromEntries(fields.map(field => [field, entity[field]])) as {
+    [F in keyof Entity]: Entity[F];
+  };
+};
+
+export const getEntitiesWithFields = (entityCodes: string[], fields: (keyof Entity)[]) =>
+  ENTITIES.map(e => e.code)
+    .filter(entityCode => entityCodes.includes(entityCode))
+    .map(entityCode => getEntityWithFields(entityCode, fields));
+
+export const sortedByCode = (entities: { code: string }[]) =>
+  entities.sort((a, b) => (a.code > b.code ? 1 : -1));
+
+export const PROJECTS = [{ code: 'redblue' }, { code: 'goldsilver' }];
+
+export const COUNTRIES = ['KANTO', 'JOHTO'];
+
+export const ENTITIES: Entity[] = [
```

Comment:
Okay plz don't judge me, but I may have modelled all the dummy data for these tests off the first two pokemon games 😂 I'm a big pokemon nerd, so it brings me great joy, but if reviewer prefers I can make it something less esoteric!

Some really nice features about this data set though. 
- In the first pokemon game, you explore a country which can easily be organised into a hierarchy. 
- In the second pokemon you explore a new country, however you can also travel back to the country which has had some changes since the first game
- This allows us to test some of the trickier corner cases with alternate hierarchies effectively (eg. what happens if an entity in one hierarchy has a different parent in another hierarchy)